examples/src/main/java/io/numaproj/numaflow/examples/function/reduce/count/Config.java

```diff
@@ -0,0 +1,11 @@
+package io.numaproj.numaflow.examples.function.reduce.count;
+
+import lombok.AllArgsConstructor;
+import lombok.Getter;
+
+@Getter
+@AllArgsConstructor
+public class Config {
+    private int oddValue;
+    private int evenValue;
```

Comment:
What's the functionality of the count reducer? I thought it's to count number of occurrences of odd/even numbers. Is it corresponding the the following reducer in numaflow-go? If so, should the Config always be oddValue = 1, evenValue = 1? When do we need to set the `Config` with customized `oddValue`/`evenValue`?

```
func reduceHandle(_ context.Context, key string, reduceCh <-chan functionsdk.Datum, md functionsdk.Metadata) functionsdk.Messages {
	// count the incoming events
	var resultKey = key
	var resultVal []byte
	var counter = 0
	for _ = range reduceCh {
		counter++
	}
	resultVal = []byte(strconv.Itoa(counter))
	return functionsdk.MessagesBuilder().Append(functionsdk.MessageTo(resultKey, resultVal))
}
```

Reply:
Just wanted to add an example of how to pass the parameters to Reducer and use it to write the aggregation logic

Reply:
Got it. please consider adding some comments on the meaning of each configuration. Better rename to something like `oddIncrementBy`?

Reply:
done